examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>This header's not rendered for some reason..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
oh weird! looking...

Reply:
it's rendering ok for my in my jupyterlab

![image](https://user-images.githubusercontent.com/7608904/103929110-63ac4f80-50e2-11eb-9c79-8f52b2bb8ce7.png)

and on GitHub https://github.com/saturncloud/examples/blob/feat/gpu-notebook-flow/examples/examples-gpu/nyc-taxi-snowflake/rf-rapids-dask.ipynb

![image](https://user-images.githubusercontent.com/7608904/103929164-7f175a80-50e2-11eb-9532-695fcbafad27.png)

So I think we should just conclude / hope it's a bug in ReviewNB